### PR TITLE
fix(cli): prepare the  string of all model settings and use it in the…

### LINKED ISSUE #2522: [CLI] Adding nested configurations to model settings
## Description / Steps to reproduce / Feature proposal
We need to add some configuration to the model settings like

```ts
@model({ 
  settings: {
    mongodb: {
      collection: 'clients',
    },
  },
})
```

## Current Behavior
In configuration.JSON I put:

```ts
"modelSettings": {
			"mongodb": {
                                 "collection": "clients"
                          }
  },
```

The CLI generated model has a string instead of JSON:

```ts
@model({ settings: { "mongodb": { "collection": "clients" } } })
```

## Expected Behavior
The generated output should be 

```ts
@model({ settings: { 
                     mongodb: { 
                            collection: "clients"
            } 
    } })
```

I think in @loopback\cli\generators\model\templates\model.ts.ejs
instead of  `JSON.stringify(modelSettings)` should be `JSON.stringify(modelSettings).replace(/\"([^(\")"]+)\":/g,"$1:")`

### PR DESCRIPTION
**Pull request by yanamg7**
… template

prepare the  string of all model settings and use it in the template, fix one test and add the new
one to test this case

fix #2522

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

### CODE REVIEW

File: packages/cli/generators/model/templates/model.ts.ejs [hunk 1]
```diff
@@ -1,35 +1,39 @@
 <% if (isModelBaseBuiltin) { -%>
-import {<%= modelBaseClass %>, model, property} from '@loopback/repository';
-<% } else { -%>
-import {model, property} from '@loopback/repository';
-import {<%= modelBaseClass %>} from '.';
-<% } -%>
+  import {<%= modelBaseClass %>, model, property} from '@loopback/repository';
```
Comment (bajtos) [inline packages/cli/generators/model/templates/model.ts.ejs]: Please revert unrelated white-space changes.

File: benchmark/package-lock.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@loopback/benchmark",
-	"version": "1.1.12",
+	"version": "1.1.13",
```
Comment (bajtos) [inline benchmark/package-lock.json]: Please revert changes in all package-lock files.

I believe we are no longer including LB4 packages in our lock files, see https://github.com/strongloop/loopback-next/commit/1d658fb6f9e0617e090adcd6265851c01ca8d375

**Comment by dhmlau** [comment]
@yanamg7 , i saw you have another PR https://github.com/strongloop/loopback-next/pull/2604, possibly for the same purpose.  Does it mean this PR should be closed?  

Please note that you don't need to open another PR for pushing more changes. Here is a detailed instruction in submitting/modifying a PR: https://loopback.io/doc/en/lb4/submitting_a_pr.html. Thanks.

**Comment by yanamg7** [comment]
Reverted and add new pull request, this one is not relevant already

**Comment by bajtos** [comment]
> Reverted and add new pull request, this one is not relevant already

In the future, please rebase your existing pull request on top of the new `master`, DO NOT open new pull requests. That way all relevant discussion stays in a single place.

In case you are not familiar with Git history rewriting, you can learn more in our docs: https://loopback.io/doc/en/contrib/code-contrib.html#how-to-rebase-your-branch.

The important part is to use `git rebase`.

**Pull request closed without merging**